torch/functional.py

```diff
@@ -392,11 +392,19 @@ def _meshgrid(*tensors):
     return _VF.meshgrid(tensors)  # type: ignore
 
 
-def stft(input, n_fft, hop_length=None, win_length=None, window=None,
-         center=True, pad_mode='reflect', normalized=False, onesided=True):
-    # type: (Tensor, int, Optional[int], Optional[int], Optional[Tensor], bool, str, bool, bool) -> Tensor
+def stft(input: Tensor, n_fft: int, hop_length: Optional[int] = None,
+         win_length: Optional[int] = None, window: Optional[Tensor] = None,
+         center: bool = True, pad_mode: str = 'reflect', normalized: bool = False,
+         onesided: Optional[bool] = None,
+         return_complex: Optional[bool] = None) -> Tensor:
     r"""Short-time Fourier transform (STFT).
 
+    The STFT computes the Fourier transform of short overlapping windows of the
+    input. This giving frequency components of the signal as they change over
+    time. The interface of this function is modelled after librosa_.
```

Comment:
I actually checked that "modelled" is OK in British English. So I guess we can keep it.

Reply:
nit: "modelled after librosa's stft function."

Reply:
Ah, I didn't realise(realize) the Americans spelt(spelled) it differently. I assume American is preferred though.